docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-managed-deploy-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='environment-managed-deploy-proc']
+= Deploying a managed environment
+
+After modifying the template for a managed environment as necessary, you must deploy the environment using the template.
+
+.Procedure
+
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the template file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc process -n <project-name> -f <template-file-name> -v ATTRIBUTE=value -V ATTRIBUTE=value... | oc create -n <project-name> -f -
+----
++
+In this command line:
++
+* Replace `<project-name>` with the name of your project
+* Replace `<template-file name>` with the full pathname of the template file that you have modified.
+* Use as many `ATTRIBUTE=value` pairs as needed to set the required attributes. You can view the template file to see descriptions for the attributes.
++
+. Set the following mandatory attributes:
+** *Application Name* (`APPLICATION_NAME`): the name of the OpenShift application. It is used in the default URLs for {CENTRAL} and {KIE_SERVER}.
```

Comment:
It is (also and more importantly) used to create a distinct set of deployment configs, services, routes, labels and artifacts and distinguish different applications built using the same or similar templates within the same project.